rust/src/mime/detect.rs

```diff
@@ -28,7 +29,7 @@ pub unsafe extern "C" fn SCDetectMimeEmailGetData(
     hname: *const std::os::raw::c_char,
 ) -> u8 {
     let c_str = CStr::from_ptr(hname); //unsafe
-    let str = c_str.to_str().unwrap_or("");
+    let str = c_str.to_str().unwrap();
```

Comment:
We should probably return 0, or -1 if the unwrap() fails to avoid the unlikely case that this could fail.

Reply:
Just don't like seeing the `.unwrap()` in live use when its easily avoidable I guess.